specification.md

```diff
@@ -1212,15 +1212,17 @@ Which would execute the mutation, creating the object and returning the followin
 }
 ```
 
-Note you can include [expressions](#Workflow-Expressions) in both `arguments` and `selectionSet`:
+Note you can include [expressions](#Workflow-Expressions) in both `arguments` 
+and `selectionSet`. The following example uses JQ string interpolation to 
+generate a valid `selectionSet` string.
 
 ```json
 {
   "refName": "getOnePet",
   "arguments": {
-    "id": "${ .petId }"
+    "id": "${.petId}"
   },
-  "selectionSet": "{ id, name, age(useDogYears: ${ .isPetADog }) { dateOfBirth, years } }"
+  "selectionSet": "${ '{ id, name, age(useDogYears: \\(.isPetADog)) { dateOfBirth, years }' }"
```

Comment:
why should be make this so hard for users to define (with \\ and ' etc)? its much easier to define the way its before

Reply:
well, this is the expression discussion all over again. I agree it would be simpler, but this doesn't compute without a clean specification for how string interpolation works.

Reply:
so, as we said to have no separate interpolation markup, that's the way strings can be created from jq.